magento/adobe-stock-integration
# #1581: Add "Edit" button to Image Details page

### Description (*)

Add's Edit button with action handler in the Image Detail's Modal.

### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1581: Add "Edit" button to Image Details page

### Manual testing scenarios (*)

1. Go to **Content - Media Gallery**
2. Select an image saved from Adobe Stock and click "three dots" in the bottom right
3. Select **View Details**
4. Verify if **Edit** button is available
5. Click Edit Button
6. Edit Details Panel should pop out
